src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -146,7 +146,7 @@ case class Bagger(
         trainingData = trainingData,
         useJackknife = useJackknife,
         biasModel = biasModel,
-        rescale = helper.ratio,
+        rescale = helper.rescaleRatio,
```

Comment:
I felt like this name is a bit better since we call this parameter `rescale` in the other parts of the code. It only involved changes in the `Bagger` file, so was not large to rename it.